File: 1. Greedy/거스름돈/restMoney.py
n = 1260
count = 0 
coint_types = [500, 100, 50 ,10]

for i in coint_types:
  #몫 만큼 더해주기
  count += n // i 
  # 해당 코인타입 나눈게 나머지 거스름돈 될거니까 :)
  n = n % i

print(count)

File: 1. Greedy/거스름돈/restMoney.ts
const n: number = 1260;
let count: number = 0;
let coin_types: number[] = [500, 100, 50, 10];

const countRest = (n: number, count: number, coin_types: number[]): number => {
  coin_types.forEach((el) => {
    count += Math.floor(n / el);
    n = n % el;
  });
  return count;
};

const result: number = countRest(n, count, coin_types);

console.log(result);
